.env.example

```
@@ -4,4 +4,5 @@ DATABASE_ID_SPRINT_SUMMARY=<Database ID of the sprint summary>
 DATABASE_ID_DAILY_SUMMARY=<Database ID of the daily summary>
 BACKLOG_PROPERTY_SPRINT=<Name of the property containing the sprint info in your product backlog>
 BACKLOG_PROPERTY_EXCLUDE_STATUS_PATTERN=<Regex of the statuses of stories which are done>
-BACKLOG_PROPERTY_STORY_POINTS=<Name of the property containing the story points in your product backlog>
\ No newline at end of file
+BACKLOG_PROPERTY_STORY_POINTS=<Name of the property containing the story points in your product backlog>
+# INCLUDE_WEEKENDS=<1 if the guideline should decrease on weekends and 0 otherwise, defaults to 1 if not specified>
```

```suggestion
INCLUDE_WEEKENDS=<true if weekends should be included in the chart, false otherwise. defaults to true>
```

would prefer true-false to 1-0 since that is more readable!

---

ok! Although will be treated as string `"true"` and `"false"` I think, but yeah more readable